tests/test_gail.py

```diff
@@ -61,7 +61,7 @@ def test_generate(generate_env):
         except TypeError:
             model = model(policy, env_name, verbose=0)
 
-    generate_expert_traj(model, 'expert', n_timesteps=1000, n_episodes=25,
```

Comment:
short info way I try to have a high `n_timesteps`.
The mentioned error with the manual env reset, only happens sometimes, so in order to adequately test this I needed a high `n_timesteps` 

The obvious problem with this is that the test then takes too long.